<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelephonesTab</class>
 <widget class="QWidget" name="TelephonesTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="homePhoneLabel">
     <property name="text">
      <string>Home Phone:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="pagerNumberLabel">
     <property name="text">
      <string>Pager Number:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="mobileNumberLabel">
     <property name="text">
      <string>Mobile Number:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="faxNumberLabel">
     <property name="text">
      <string>Fax Number:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="iPPhoneNumberLabel">
     <property name="text">
      <string>IP Phone Number:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="notesLabel">
     <property name="text">
      <string>Notes:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPlainTextEdit" name="notes_edit"/>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="home_phone_edit"/>
     </item>
     <item>
      <widget class="QPushButton" name="home_phone_button">
       <property name="text">
        <string>Other...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="pager_edit"/>
     </item>
     <item>
      <widget class="QPushButton" name="pager_button">
       <property name="text">
        <string>Other...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="mobile_edit"/>
     </item>
     <item>
      <widget class="QPushButton" name="mobile_button">
       <property name="text">
        <string>Other...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="fax_edit"/>
     </item>
     <item>
      <widget class="QPushButton" name="fax_button">
       <property name="text">
        <string>Other...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLineEdit" name="ip_phone_edit"/>
     </item>
     <item>
      <widget class="QPushButton" name="ip_phone_button">
       <property name="text">
        <string>Other...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
